examples/button/button/app.py

```diff
@@ -117,6 +140,32 @@ def callbackSmaller(self, button):
         #   In this case the window size will change
         self.main_window.size = (200, 200)
 
+    def seek_origin(self, button):
+        # Move the window to the somewhere near the top-left of the main screen
+        self.main_window.position = (100, 30)
+
+    def seek_right_screen(self, button):
+        # Move the window off the right hand side of the main screen.
+        # If the user has multiple monitors with a screen to the right
+        # of the main screen, the window should appear there.
+        self.main_window.position = (2000, 400)
```

Comment:
There is no guarantee that this actually gives us the right screen since the primary screen may have a width > 2000 pixels.

Reply:
Completely agreed. 2000 "works for me", but that's hardly reliable. Ideally, we'd be able to get reliable screen reference information as part of the Toga API... but this yak is already sporting a magnificent hairstyle... :-)